frontend/src/pages/ProgressionChecker/GridView/GridModal/styles.jsx

```diff
@@ -0,0 +1,47 @@
+import { Modal, Typography } from "antd";
+import styled from "styled-components";
+import GRID from "../styles";
+
+const { Title } = Typography;
+
+const ModalHeader = styled.div`
+  padding-top: 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+`;
+
+const ModalTitle = styled(Title)`
+  && {
+    margin: 0 !important;
+  }
+`;
+
+const Instruction = styled.p`
+  margin-top: 10px;
+  margin-bottom: 0px;
+`;
+
+const CourseModal = styled(Modal)`
+  top: 0px;
+  margin: 30px;
+  float: right;
+  .ant-modal-content, .ant-modal-header {
```

Comment:
after looking at the modal, think its better to remove the border-radius styling 😅